validate/validate.go

```diff
@@ -337,11 +337,11 @@ func (v *Validator) CheckRlimits() (msgs []string) {
 func supportedMountTypes(OS string, hostSpecific bool) (map[string]bool, error) {
 	supportedTypes := make(map[string]bool)
 
-	if OS != "linux" && OS != "windows" {
+	if OS != "linux" && OS != "solaris" {
 		logrus.Warnf("%v is not supported to check mount type", OS)
 		return nil, nil
-	} else if OS == "windows" {
-		supportedTypes["ntfs"] = true
+	} else if OS == "solaris" {
+		supportedTypes["lofs"] = true
```

Comment:
This is not correct modification.  You can refer to FSTYPE in [mount(1)](http://docs.oracle.com/cd/E86824_01/html/E54764/mount-1m.html#REFMAN1Mmount-1m) to determine solaris supports fs types.
Windows does not support type in mount, there is no need to output warning and we should accept this change after spec v1.0 released.